src/contact.ts

```diff
@@ -477,6 +477,37 @@ export class Contact implements Sayable {
    */
   public alias(empty: null): Promise<boolean>
 
+  /**
+   * GET / SET / DELETE the alias for a contact
+   *
+   * @param {(none | string | null)} newAlias ,
+   * @returns {(string | null | Promise<boolean>)}
+   *
+   * @example GET the alias for a contact
+   * ```ts
+   * const alias = contact.alias()  //@returns {(string | null)}
+   * ```
+   *
+   * @example SET the alias for a contact
+   * ```ts
+   * const ret = await contact.alias('lijiarui')
+   * if (ret) {
+   *   console.log(`change ${contact.name()}'s alias successfully!`)
+   * } else {
+   *   console.error('failed to change ${contact.name()}'s alias!')
+   * }
+   * ```
+   *
+   * @example DELETE the alias for a contact
+   * ```ts
+   * const ret = await contact.alias(null)
+   * if (ret) {
+   *   console.log('ok!')
+   * } else {
+   *   console.error('fail!')
+   * }
+   * ```
```

Comment:
Maybe here should be changed as follows, it would better than I wrote before:
```ts
const ret = await contact.alias(null)
if (ret) {
  console.log(`delete ${contact.name()}'s alias successfully!`)
} else {
  console.log(`failed to delete ${contact.name()}'s alias!`)
}
```